### PR TITLE
Used x as template log fixed to have the templates title.

### PR DESCRIPTION
**Pull request by hmoco**
https://openscience.atlassian.net/browse/OSF-4926

### CODE REVIEW

**Comment by chennan47** [comment]
CR1 done. Looks good.

**Comment by samanehsan** [comment]
CR2 looks good to me @hmoco :balloon:

**Comment by sloria** [comment]
@hmoco What will happen with old logs that don't have title in `params`?

**Comment by hmoco** [comment]
Good call, it will show blank, working on a solution right now

**Comment by hmoco** [comment]
@sloria, would a migration be preferable, or a way to display ` another ` if title is not available?

**Comment by sloria** [comment]
@hmoco There is no perfect way to know what the title of a node is at the the time a log was created (the title may have changed over time).  So for now, just handle both log formats. No migration.

File: website/templates/log_templates.mako [hunk 1]
```diff
@@ -85,7 +85,7 @@ deleted
 
 <script type="text/html" id="created_from">
 created
-<a class="log-node-title-link overflow" data-bind="text: nodeTitle, attr: {href: nodeUrl}"></a> based on <a class="log-node-title-link overflow" data-bind="attr: {href: params.template_node.url}">another</a>
+<a class="log-node-title-link overflow" data-bind="text: nodeTitle, attr: {href: nodeUrl}"></a> based on <a class="log-node-title-link overflow" data-bind="text: params.template_node.title ? params.template_node.title : 'another', attr: {href: params.template_node.url}"></a>
```
Comment (sloria) [inline website/templates/log_templates.mako]: Minor: You can simplify this to 

```js
params.template_node.title || 'another'
```

I will make this change when I merge.

**Comment by sloria** [comment]
Merged into develop

**Pull request closed without merging**